src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -918,6 +918,25 @@ executing the TerminateProcess method of the process. Whereas the SIGKILL is equ
 the forceful removal of the running process.
 
 
+===== Termination Behavior by Language
+
+.Termination Behavior by Language
+|===
+|Language/Execution type|Bash |Python
```

Comment:
That's correct, an InterruptedException will be thrown in Java or groovy in non-forked mode, and a shutdownHook could be added in forked mode to handle the SIGTERM signal